src/TinyIoC/TinyIoC.cs

```diff
@@ -3938,17 +3960,8 @@ public void Dispose()
         #endregion
     }
 
-}
-
 // reverse shim for WinRT SR changes...
 #if !NETFX_CORE
-namespace System.Reflection
-{
-#if TINYIOC_INTERNAL
-    internal
-#else
-    public
-#endif

```

Comment:
I internalized this inside of the tiny ioc namespace.. This prevents it leaking into other libraries like json.net (This happened to me and I couldn't compile as we also internalize json.net)